<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xWidgetsClass</class>
 <widget class="QWidget" name="xWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="titleHorizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QWidget" name="titleBarWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="icon_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-image: url(:/icons/desktop.ico);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="title_label">
          <property name="text">
           <string>标题</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="min_button">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  margin: 4px 8px 4px 8px;
  border-image: url(:/icons/window_icons/md-min.png);
}
QPushButton:hover {
  background-color: rgba(85, 170, 255, 0.5);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="max_button">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  margin: 4px 8px 4px 8px;
  border-image: url(:/icons/window_icons/md-restore.png);
}
QPushButton:hover {
  background-color: rgba(85, 170, 255, 0.5);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_button">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  margin: 4px 8px 4px 8px;
  border-image: url(:/icons/window_icons/md-close.png);
}
QPushButton:hover {
  background-color: rgba(255, 0, 0, 128);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="centralVerticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="centralWidget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resource/EasyMeasure.qrc"/>
 </resources>
 <connections/>
</ui>
